src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -202,15 +202,15 @@ def validate_load_balancer_outbound_ports(namespace):
     if namespace.load_balancer_outbound_ports is not None:
         if namespace.load_balancer_outbound_ports % 8 != 0:
             raise CLIError("--load-balancer-allocated-ports must be a multiple of 8")
-        if namespace.load_balancer_outbound_ports < 0:
-            raise CLIError("--load-balancer-allocated-ports cannot be negative")
+        if namespace.load_balancer_outbound_ports < 0 or namespace.load_balancer_outbound_ports > 64000:
```

Comment:
you know this number is based on number of VMs and number of FE IP, right? is it what we have implemented in RP? I guess it's better than before...

Reply:
Yup, that is what we implemented in RP. 

We do a check that: IPs * 64K > # of vms * desired allocated ports 

where # of vms accounts for things like buffer for upgrading and over provisioning, auto scaler etc.